diff --git a/node_modules/@microsoft/tui-test/lib/runner/runner.js b/node_modules/@microsoft/tui-test/lib/runner/runner.js
index 96eed8a..52e753a 100644
--- a/node_modules/@microsoft/tui-test/lib/runner/runner.js
+++ b/node_modules/@microsoft/tui-test/lib/runner/runner.js
@@ -77,7 +77,8 @@ const checkRuntimeVersion = () => {
         nodeMajorVersion != "18" &&
         nodeMajorVersion != "20" &&
         nodeMajorVersion != "22" &&
-        nodeMajorVersion != "24") {
+        nodeMajorVersion != "24" &&
+        nodeMajorVersion != "25") {
         console.warn(chalk.yellow(`Warning: ${executableName} works best when using a supported node versions (which ${nodeVersion} is not).\n`));
     }
 };
diff --git a/node_modules/@microsoft/tui-test/lib/trace/viewer.js b/node_modules/@microsoft/tui-test/lib/trace/viewer.js
index 46621d1..80b4f8e 100644
--- a/node_modules/@microsoft/tui-test/lib/trace/viewer.js
+++ b/node_modules/@microsoft/tui-test/lib/trace/viewer.js
@@ -4,11 +4,16 @@ import readline from "node:readline";
 import { EventEmitter } from "node:events";
 import chalk from "chalk";
 import ansi from "../terminal/ansi.js";
-const rl = readline.createInterface({
-    input: process.stdin,
-    output: process.stdout,
-});
-const question = (query) => new Promise((resolve) => rl.question(query, (answer) => resolve(answer)));
+const question = (query) => {
+    const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout,
+    });
+    return new Promise((resolve) => rl.question(query, (answer) => {
+        rl.close();
+        resolve(answer);
+    }));
+};
 export const play = async (trace) => {
     const startTime = trace.tracePoints.find((tracePoint) => "time" in tracePoint).time;
     const startSize = trace.tracePoints.find((tracePoint) => "rows" in tracePoint);
